src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/SignAndAuthKeys.vue

```diff
@@ -120,25 +125,34 @@ export default Vue.extend({
         return 0;
       });
 
-      if (!this.search) {
+      // Check that there is a search input
+      if (!this.search || this.search.length < 1) {
         return arr;
       }
 
       const mysearch = this.search.toLowerCase();
 
-      if (mysearch.length < 1) {
-        return this.tokens;
-      }
-
       arr.forEach((token: Token) => {
         token.keys.forEach((key: Key) => {
+          // Filter the certificates
           const certs = key.certificates.filter((cert: TokenCertificate) => {
             if (cert.owner_id) {
               return cert.owner_id.toLowerCase().includes(mysearch);
             }
             return false;
           });
           key.certificates = certs;
+
+          // Filter the CSR:s
+          const csrs = key.certificate_signing_requests.filter(
+            (csr: TokenCertificateSigningRequest) => {
+              if (csr.id) {
+                return csr.id.toLowerCase().includes(mysearch);
+              }
+              return false;
+            },
```

Comment:
Should it be possible to filter with word "Request" , which is visible in entries for  CSR:s?  If you want to see only CSR:s...

Reply:
In practice it should filter with the localised word for "Request". Seems to me a bit too complicated system for a very rare use case. But I'll mention this in a sprint review.